packages/perseus-core/src/widgets/core-widget-registry.ts

```diff
@@ -78,11 +80,41 @@ export function isAccessible(
     type: string,
     widgetOptions: PerseusWidgetOptions,
 ): boolean {
-    const accessible = widgets[type]?.accessible;
+    const accessible = widgets.get(type)?.accessible;
     return typeof accessible === "function"
         ? accessible(widgetOptions)
         : !!accessible;
 }
+
+export const traverseChildWidgets = (
+    widgetInfo: PerseusWidget,
+    traverseRenderer: any,
+): PerseusWidget => {
+    if (!traverseRenderer) {
+        throw new PerseusError(
+            "traverseRenderer must be provided, but was not",
+            Errors.Internal,
+        );
+    }
+
+    // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+    if (!widgetInfo || !widgetInfo.type || !widgets[widgetInfo.type]) {
+        return widgetInfo;
+    }
+
+    const widgetExports = widgets[widgetInfo.type];
+    const props = widgetInfo.options;
+
+    if (widgetExports.traverseChildWidgets != null && props != null) {
```

Comment:
This should be using the `.get()` function instead of an indexer(`[]`). Once you do this, you'll see that we need to adjust the check too slightly. 

```suggestion
    const widgetExports = widgets.get(widgetInfo.type);
    const props = widgetInfo.options;

    if (widgetExports?.traverseChildWidgets != null && props != null) {
```

Reply:
Thanks for pointing that out! I've updated it to use .get()